src/main/java/org/isf/medicalstockward/service/MedicalStockWardIoOperations.java

```diff
@@ -246,6 +246,7 @@ protected boolean updateStockWardQuantity(
 				medicalWard.setLot(movement.getLot());
 				repository.save(medicalWard);
             }
+	        repository.updateOutQuantity(Math.abs(qty), ward, medical, lot);
```

Comment:
maybe indentation?

Reply:
Fixed the indentation and while I was at it corrected some formatting in that area too.

Reply:
saw it, thanks!